cdap-api/src/main/java/co/cask/cdap/api/common/exceptions/HTTPResponseException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.common.exceptions;
+
+/**
+ * CDAP HTTP Response Exception
+ */
+public interface HttpResponseException {
```

Comment:
The name of the exception class should represent what is causing the exception and the exception is not caused by a response. It is actually caused by the request.